java/src/org/openqa/selenium/edge/EdgeDriverService.java

```diff
@@ -248,16 +231,24 @@ public Builder withLoglevel(ChromiumDriverLogLevel logLevel) {
      * @return A self reference.
      */
     public Builder withSilent(boolean silent) {
-      if (silent) {
-        this.logLevel = ChromiumDriverLogLevel.OFF;
-      }
```

Comment:
I think you need to assign the value to `this.silent`, which is what you did in the `ChromeDriverService`.